packages/editor/src/components/post-card-panel/index.js

```diff
@@ -27,55 +27,47 @@ import {
 } from '../../store/constants';
 import { unlock } from '../../lock-unlock';
 
-export default function PostCardPanel( { actions } ) {
+export default function PostCardPanel( { postType, postId, actions } ) {
 	const { isFrontPage, isPostsPage, title, icon, isSync } = useSelect(
 		( select ) => {
-			const {
-				getEditedPostAttribute,
-				getCurrentPostType,
-				getCurrentPostId,
-				__experimentalGetTemplateInfo,
-			} = select( editorStore );
-			const { canUser } = select( coreStore );
-			const { getEditedEntityRecord } = select( coreStore );
+			const { __experimentalGetTemplateInfo } = select( editorStore );
+			const { canUser, getEditedEntityRecord } = select( coreStore );
 			const siteSettings = canUser( 'read', {
 				kind: 'root',
 				name: 'site',
 			} )
 				? getEditedEntityRecord( 'root', 'site' )
 				: undefined;
-			const _type = getCurrentPostType();
-			const _id = getCurrentPostId();
-			const _record = getEditedEntityRecord( 'postType', _type, _id );
+			const _record = getEditedEntityRecord(
+				'postType',
+				postType,
+				postId
+			);
 			const _templateInfo =
 				[ TEMPLATE_POST_TYPE, TEMPLATE_PART_POST_TYPE ].includes(
-					_type
+					postType
 				) && __experimentalGetTemplateInfo( _record );
 			let _isSync = false;
-			if ( GLOBAL_POST_TYPES.includes( _type ) ) {
-				if ( PATTERN_POST_TYPE === _type ) {
+			if ( GLOBAL_POST_TYPES.includes( postType ) ) {
+				if ( PATTERN_POST_TYPE === postType ) {
 					// When the post is first created, the top level wp_pattern_sync_status is not set so get meta value instead.
 					const currentSyncStatus =
-						getEditedPostAttribute( 'meta' )
-							?.wp_pattern_sync_status === 'unsynced'
+						_record?.meta?.wp_pattern_sync_status === 'unsynced'
```

Comment:
`getEditedPostAttribute` does a lot more things than just returning the att. With the changes here, the postcard would only be updated _after_ saving the changes while before it was updated as they changes happen (change title in panel). Is that correct? Is it the reason for these changes?

Reply:
I've noticed this: 

https://github.com/user-attachments/assets/0514216f-af2c-4ebf-a333-9ed8aa72b784

Reply:
If we show the edited entity in the editor (title, post card), shouldn't we do the same in dataviews if we aim to provide a seamless editing experience?

Reply:
> getEditedPostAttribute does a lot more things than just returning the att. With the changes here, the postcard would only be updated after saving the changes while before it was updated as they changes happen (change title in panel). Is that correct? Is it the reason for these changes?

the `record` is computed using `getEditedEntityRecord` (notice the "edited" in the name) so it's also going to bring the "unsaved edits". The reason for this change is that I don't want to rely on the "editor" store. I can call "core" directly and we're already doing so using `getEditedEntityRecord`

Reply:
> If we show the edited entity in the editor (title, post card), shouldn't we do the same in dataviews if we aim to provide a seamless editing experience?

Yes, this seems like a separate bug IMO. But it also relates to the "update" button. I guess we should remove the "update" button in this case and just rely on the global save button.